var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -16,7 +16,10 @@ class Cube(AutotoolsPackage):
 
     homepage = "https://www.scalasca.org/software/cube-4.x/download.html"
     url = "https://apps.fz-juelich.de/scalasca/releases/cube/4.4/dist/cubegui-4.4.2.tar.gz"
+    maintainers = "swat-jsc"
```

Comment:
This syntax is no longer supported.  You need to use the new directive described at docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers.